code/datums/components/food/germ_carrier.dm

```diff
@@ -0,0 +1,100 @@
+// Don't eat off the floor or hold parent object with dirty hands, you'll get sick
+
+/// Time needed for bacteria to infect the parent object
+#define FIVE_SECOND_RULE (5 SECONDS)
```

Comment:
actual variable name, especially since i dont think this should be just 5 seconds anyway